mockingbird/src/commonTest/kotlin/com/careem/mockingbird/test/FunctionsTest.kt

```diff
@@ -159,6 +159,25 @@ class FunctionsTest {
         assertEquals(TEST_STRING, stringSlot.captured)
     }
 
+    //TODO verify that the slot is a LocalThreadSlot
```

Comment:
You can verify this by checking that captured value is not frozen.

Reply:
Good idea 🙏